20-packages.ks

```diff
@@ -11,29 +16,23 @@ chkconfig
 rootfiles
 yum
 vim-minimal
-acpid
-tftp
```

Comment:
TFTP utility is needed to download extensions, is that not available anymore in EL8?